src/Compilers/CSharp/Test/Emit2/Semantics/CollectionExpressionTests.cs

```diff
@@ -35272,20 +35316,20 @@ static void Main()
                     {
                         IEnumerable<int> e = [1, 2, 3];
                         e.Report();
-                        List<int> list = [..e];
+                        List<int> list = [..e, 4];
```

Comment:
I didin't want to mix single-spread and `AddRange` tests together and I wanted to be sure, that `AddRange` code paths are still being verified. Therefore I modified existing `AddRange` tests, but added an equivalent `SingleSpread` tests on top for these cases